### PR TITLE
Add cosapp_lab recipe

### PR DESCRIPTION
**Pull request by trungle-safran**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by trungle-safran** [comment]
Hello @conda-forge/staged-recipes, this recipe is ready for review.

File: recipes/cosapp_lab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  entry_points:
+    - cosapp = cosapp_lab.main:cosapp
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+
+  build:
+    - setuptools
+    - jupyter-packaging
+
```
Comment (chrisburr) [inline recipes/cosapp_lab/meta.yaml:26]: Is `build` the right place for these? I think I would expect to see them in `host`

File: recipes/cosapp_lab/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  sha256: f7c1b9a9466e2d64792b7e1b5933af1b6b3af5d402b9c83b5e2be00da9658680
+  
+build:
+  noarch: python
+  number: 0
+  skip_compile_pyc:
+    - "*/utils/templates/*.py" 
+  entry_points:
+    - cosapp = cosapp_lab.main:cosapp
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (chrisburr) [inline recipes/cosapp_lab/meta.yaml:19]: These options are the default when using conda-build
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv"
```

File: recipes/cosapp_lab/meta.yaml (hunk 1 above)
Comment (trungle-safran) [reply recipes/cosapp_lab/meta.yaml]: Thank you for your review, updated

File: recipes/cosapp_lab/meta.yaml (hunk 2 above)
Comment (trungle-safran) [reply recipes/cosapp_lab/meta.yaml]: updated

**Comment by trungle-safran** [comment]
@conda-forge-admin, please ping team

**Pull request merged**